### PR TITLE
feat: FAB for add resource/review

### PR DESCRIPTION
**Pull request by proinators**
## Description

<pre>
Minor change to the UI: FAB in the course review/resources page

For smaller screens, the 'Add resource'/'Add review' button is replaced with a floating action button
</pre>

## Type of change

- [x] New website feature (non-breaking change which adds functionality)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -60,4 +60,6 @@ venv/
 .vscode/
 
 # Logs
-**.log
\ No newline at end of file
+**.log
+
+.env
```
Comment (thenicekat) [inline .gitignore:65]: shouldn't you use .env.local anyways for next js?

File: pages/courses/resources/index.tsx [hunk 2]
```diff
@@ -92,13 +92,18 @@ export default function Resources() {
...
-                            <div className="flex flex-col md:flex-row w-1/3 justify-center">
+                            <div className="flex flex-col hidden md:block md:flex-row w-1/3 justify-center">
                                 <Link className="m-3 w-full" href={"/courses/resources/add"}>
                                     <button className="btn btn-outline w-full">
                                         Add a Resource
                                     </button>
                                 </Link>
                             </div>
+                            <div className="z-10 w-14 fixed bottom-5 right-0 m-4 cursor-pointer md:hidden">
+                                <Link className="m-3 w-full" href={"/courses/resources/add"}>
```
Comment (thenicekat) [inline pages/courses/resources/index.tsx:103]: what is the color of the plus? once check rants page, just in case

File: .gitignore (hunk 1 above)
Comment (proinators) [reply .gitignore]: Yes, corrected now

File: pages/courses/resources/index.tsx (hunk 2 above)
Comment (proinators) [reply pages/courses/resources/index.tsx:103]: Earlier it was grey, but yes, it looks bad
Changed to cyan, like the rants page now

**Review by thenicekat — APPROVED** [review]

**Pull request merged**